python/gui/auto_generated/qgsprovidersublayersdialog.sip.in

```diff
@@ -56,6 +56,7 @@ Dialog for selecting provider sublayers.
   public:
 
     QgsProviderSublayersDialog( const QString &uri,
+                                const QString &providerKey,
```

Comment:
Sorry - I should have picked this one up earlier, but adding this argument here is an API break. Can you add it as an optional argument at the end to avoid this?

Reply:
Whoops I see this class was introduced in 3.30... so ignore my comment, still on holiday brain!